lib/side_menu_animation.dart

```diff
@@ -11,6 +11,8 @@ typedef SideMenuAnimationAppBarBuilder = AppBar Function(VoidCallback showMenu);
 const _sideMenuWidth = 100.0;
 const _sideMenuDuration = const Duration(milliseconds: 800);
 
+enum Position { rigth, left }
```

Comment:
Rename `Position` to `SideMenuPosition` to avoid conflicts with other packages/code.